transmart-core-api/src/main/groovy/org/transmartproject/core/users/PatientDataAccessLevel.groovy

```diff
@@ -14,6 +14,11 @@ enum PatientDataAccessLevel {
      */
     COUNTS_WITH_THRESHOLD,
 
+    /**
+     * Only access to patient counts
+     */
+    COUNTS,
```

Comment:
Other levels are described here: https://github.com/thehyve/transmart-core/tree/dev/transmart-data/updatedb/release-17.1-HYVE-5#migrate-access-levels. Maybe this one could be mentioned somewhere as well?